build-android/README.md

```diff
@@ -2,18 +2,48 @@
 
 Download the latest [build for Android](http://d.etpan.org/mailcore2-deps/mailcore2-android/)
 
-### Build for Android ###
+### Build on Mac ###
+Tested with macOS 10.13.6, Android Studio 3.1.4
 
+Install Java 6, 8, or 10
+* http://www.oracle.com/technetwork/java/javase/downloads/jdk10-downloads-4416644.html
+* Building with Java 8 or 10 might throw a few warnings during the build, but it will still build
+
+Download the Required Android SDKs (16, 21, 22)
+* Download using Android Studio
+
+Download the Required NDK (13)
+* https://dl.google.com/android/repository/android-ndk-r13b-darwin-x86_64.zip
+
+Open terminal and set two temporary export variables
+* (Make sure they point to your SDK and NDK folders)
+```
+export ANDROID_SDK=/Users/YOURNAME/Library/Android/sdk
```

Comment:
Could you keep it as /path/to/android-sdk (and maybe provide a specific example).

Like

```
$ export ANDROID_SDK=/path/to/android-sdk # /Users/username/Library/Android/sdk
```